### PR TITLE
[TRAFODION-2857] Web-site changes to remove incubating status

### PR DESCRIPTION
**Pull request by svarnau**
Project status and URL changes.
Email lists and git repo name changes still to come based on infra changes.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -13,16 +13,16 @@ To get started using Apache Trafodion:
...
   - documentation for this release can be viewed on the Apache Trafodion 
     wiki page: 
-      http://trafodion.incubator.apache.org/documentation.html
+      http://trafodion.apache.org/documentation.html
 
   - release notes describing changes for particular releases can be viewed:
       http://trafodion.apache.org/release-notes.html
 
-  - the latest Apache Trafodion can be downloaded from Apache incubator
+  - the latest Apache Trafodion can be downloaded from Apache
```
Comment (DaveBirdsall) [inline README]: Grammar nit: Do you want to say "the Apache distribution site" (add "the") ?

File: docs/client_install/src/asciidoc/_chapters/preparation.adoc [hunk 2]
```diff
@@ -220,7 +219,7 @@ $ unzip trafci.zip -d $HOME/trafodion/trafci
 .
 $ cd ..
 $ ls
-apache-trafodion_clients-2.0.1-incubating.tar.gz  clients  jdbct4  trafci
+apache-trafodion_clients-2.2.0.tar.gz  clients  jdbct4  trafci
```
Comment (DaveBirdsall) [inline docs/client_install/src/asciidoc/_chapters/preparation.adoc]: Release 2.2 hasn't been officially approved yet... should this say, 2.1.x?

Comment (svarnau) [reply docs/client_install/src/asciidoc/_chapters/preparation.adoc]: This is example in documentation on master branch, for next release. Not for past releases that were incubating releases.

File: README.md [hunk 3]
```diff
@@ -20,11 +20,11 @@ Hadoop.
   
   http://trafodion.apache.org/release-notes.html
 
-  - the latest Apache Trafodion can be downloaded from Apache incubator
+  - the latest Apache Trafodion can be downloaded from Apache
     distribution site or accessed from git:
 
     - download site:
-      https://dist.apache.org/repos/dist/release/incubator/trafodion
+      https://dist.apache.org/repos/dist/release/trafodion
```
Comment (mkby) [inline README.md]: currently this URL cannot be found...

File: README (hunk 1 above)
Comment (svarnau) [reply README]: Okay, I will add "the"

File: README.md (hunk 3 above)
Comment (svarnau) [reply README.md]: Good point. I was anticipating this change.  I thought I could move the dist site with a couple svn commands, but I lack permission, so I have inquired with infrastructure team.

**Comment by sandhyasun** [comment]
+1

File: README.md (hunk 3 above)
Comment (svarnau) [reply README.md]: Infra team has taken care of moving the dist directory.

**Pull request merged**